tensorboard/components/tf_dashboard_common/data-loader-behavior.ts

```diff
@@ -129,9 +129,9 @@ namespace tf_dashboard_common {
     },
 
     created() {
-      this._loadData = _.debounce(this._loadDataImpl, 100, {
+      this._loadData = _.throttle(this._loadDataImpl, 100, {
         leading: true,
-        trailing: true,
+        trailing: false,
```

Comment:
:( you really want to invoke the method on trailing as well. If `dataToLoad` as changed, you want to fetch the ones that are new.